lnp2p/peermgr.go

```diff
@@ -141,11 +137,12 @@ func (pm *PeerManager) GetPeerByIdx(id int32) *Peer {
 	return pm.peerMap[pm.peers[id]]
 }
 
-// TryConnectAddress attempts to connect to the specified LN address.
+// TryConnectAddress is a handler function for tryConnectPeer and
```

Comment:
I'm not sure I understand what you mean by "is a handler function" here.  This tries to hide the details of calling `tryConnectPeer`.

Reply:
Hm, I don't think this is a "strict" change, so I can remove this